features/product/viewing_product_variants/viewing_list_of_product_variants_with_corresponding_prices_and_options.feature

```diff
@@ -0,0 +1,28 @@
+@viewing_product_variants
```

Comment:
Why did you put these scenarions in a separate file instead viewing_products/viewving_different_price_for_different...?

Reply:
This is because in the API context the product does not contain price information (unlike the UI), so in my opinion `@viewing_product_variants` fits better here